src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,202 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+interface DayOverviewerPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+}
+
+interface DayOverviewerCardProps extends DayOverviewerPdfProps {}
```

Comment:
Can you explain why we need this interface?
(Seems like we can just replace the usage below with `DayOverviewedPdfProps`)

Reply:
(the comment is outdated for some reason so i'll just resolve this and re-comment on the latest commit)